### DISCUSSION TITLE
Have you considered a JOSS submission?

### DISCUSSION THREAD

**Discussion by sgbaird** [emdgroup/baybe#84] [Ideas]
It's a nice journal to highlight packages like this. This could be a good way to get additional visibility and community involvement as well as get some critical feedback from reviewers. The write up is quite short, and I've had a good experience with this journal in the past.

https://joss.theoj.org/

**Comment by Scienfitz** [comment]
Hi @sgbaird,

excellent idea, we are anyway currently drafting up some stuff possibly to be submitted to some journal (will still take some more time though). 

It seems the publications at JOSS are fairly lightweight, so I can easily see us change and slim down what we have once the draft is finished and publish at JOSS too

**Comment by Scienfitz** [comment]
outdated

as our [publication in Digital Discovery](https://pubs.rsc.org/en/content/articlelanding/2025/dd/d5dd00050e) is live we would rather plan to support this via future commit publications rather than a separate JOSS publication
